lib/ssm_storage/ssm_storage_db.rb

```diff
@@ -0,0 +1,76 @@
+class SsmStorageDb
+  TABLE_NAME = 'ssm_config_records'.freeze
+  ACTIVE_RECORD_MODEL = 'SsmConfigRecord'.freeze
+  def initialize(file_name)
+    @file_name = file_name
+  end
+
+  def file_exists?
+    active_record_exists = ActiveRecord::Base.connection.table_exists? TABLE_NAME
+    if active_record_exists && (defined? (ACTIVE_RECORD_MODEL.constantize) == 'constant')
```

Comment:
`Object.const_defined?(ACTIVE_RECORD_MODEL)` will do the trick here.
You might be able to refactor this to remove the if/else and just return the 3 part boolean expression with `&&`.